src/main/java/com/monitorjbl/xlsx/impl/StreamingSheetReader.java

```diff
@@ -89,8 +91,22 @@ private void handleEvent(XMLEvent event) throws SAXException {
       String tagLocalName = startElement.getName().getLocalPart();
 
       if("row".equals(tagLocalName)) {
-        Attribute rowIndex = startElement.getAttributeByName(new QName("r"));
-        currentRow = new StreamingRow(Integer.parseInt(rowIndex.getValue()) - 1);
+        Attribute rowNumAttr = startElement.getAttributeByName(new QName("r"));
+        Attribute isHiddenAttr = startElement.getAttributeByName(new QName("hidden"));
+        int rowIndex = Integer.parseInt(rowNumAttr.getValue()) - 1;
+        boolean isHidden = isHiddenAttr != null && "1".equals(isHiddenAttr.getValue());
+        currentRow = new StreamingRow(rowIndex, isHidden);
+      } else if("col".equals(tagLocalName)) {
+        Attribute isHiddenAttr = startElement.getAttributeByName(new QName("hidden"));
+        boolean isHidden = isHiddenAttr != null && "1".equals(isHiddenAttr.getValue());
+        if (isHidden) {
+          Attribute minAttr = startElement.getAttributeByName(new QName("min"));
+          Attribute maxAttr = startElement.getAttributeByName(new QName("max"));
+          int min = Integer.parseInt(minAttr.getValue()) - 1;
+          int max = Integer.parseInt(maxAttr.getValue()) - 1;
+          for (int columnIndex = min; columnIndex <= max; columnIndex++)
+            hiddenColumns.add(columnIndex);
```

Comment:
This is in support of org.apache.poi.ss.usermodel.Sheet.isColumnHidden(int columnIndex) method, not the org.apache.poi.ss.usermodel.Row.getZeroHeight() method.